packages/lab/src/Flow/base.ts

```diff
@@ -232,13 +232,8 @@ export const flowStyles = css`
   .react-flow__node-input,
   .react-flow__node-output,
   .react-flow__node-group {
```

Comment:
This style was actually needed when showing an invalid node:

<img width="199" alt="Screenshot 2024-08-20 at 11 54 01" src="https://github.com/user-attachments/assets/adbd98ce-ee92-4b76-a93d-3cd3e066033d">

We should recover this:

```css
  .react-flow__node-default,
  .react-flow__node-input,
  .react-flow__node-output {
     padding: ${theme.space.sm};
     border-radius: ${theme.radii.round};
     width: 150px;
     color: ${theme.colors.secondary};
     text-align: center;
     border: 1px solid ${theme.colors.negative};
     background-color: ${theme.colors.negative_20};
  }
  .react-flow__node-group {
    color: ${theme.colors.secondary};
    text-align: center;
  }
```